### ISSUE TITLE
a11y: modal aria-label

### ISSUE THREAD

**Issue by bellsworth13** [carbon-design-system/carbon-components-react#733]
## a11y: modal aria-label: To help users navigate a page, WAI-ARIA widgets must have an accessible name specified with aria-label, aria-labelledby or the widget's inner text. 

## Detailed description

![image](https://user-images.githubusercontent.com/17937450/37676705-a28f1b42-2c46-11e8-918c-e73fd792799b.png)

**Steps to reproduce**
1) Login IBM Could with IBM ID and Password
2) Click user icon at top right
3) Click Cloud Foundry Organizations
4) Select one user, Click Rename under the menu

**Expected behavior**
No error report in DAP

**Comment by tw15egan** [comment]
To fix: The component should have an `ariaLabel` prop that can set the `aria-label` attribute
